### DISCUSSION TITLE
Edge margins

### DISCUSSION THREAD

**Discussion by Depechie** [UI-Lovelace-Minimalist/UI#946] [Q&A] [answered]
Can someone tell me how to minimize the edge margins? What file and value to change?
I want the cards more to the side of the screen because some card titles are a bit longer and it would be nice to have more room.

I know currently this means readjusting the file/value after each upgrade because it is not something we can configure, but I do not mind that.

**Comment by AndyVRD** [comment, marked as answer]
Remove this part from your selected theme file(s).

**Theme files location:** custom_components/ui_lovelace_minimalist/lovelace/themefiles

```
  card-mod-theme: "minimalist-desktop"
  card-mod-view-yaml: |
    "*:first-child$": |
      #columns .column > * {
        padding-left: 5px;
        padding-right: 5px;
        padding-bottom: 5px;
      }
```
